lib/bootbasetest.pm

```diff
@@ -10,19 +10,19 @@ sub post_fail_hook {
         record_info 'Seems that the display manager failed';
     }
 
+    # crosscheck for text login on tty1
+    select_console 'root-console';
```

Comment:
Doing this here means that we can never reach the lines below with `match_has_tag`. Looks like back-and-forth for me. Rather I recommend to foresee all alternatives and act accordingly. During "boot" we can end up in multiple situations, a getty prompt is just one of them

Reply:
@okurz 
Why could not we reach `match_has_tag`? We can use it a bit latter in the code, no?
And we switch to `root-console` anyway before `save_memory_dump`.

We can have the following scenarios in the post-fail hook:
* screen is black because gfx is broken => switch to tty and export logs and dump memory
* screen is black because system hanged => unable to switch to tty, you cannot dump memory either
* screen shows unexpected desktop

So, I would remove all the screen check and just switch to a tty, export logs and dump memory on qemu only in a final step. WDYT?

Reply:
I see (part of) the problem now: There is a `select_console` before `save_memory_dump`. I don't know what that is there but it clearly does not make sense. If you want to continue here I think that select_console should be removed